Source/JavaScriptCore/assembler/LinkBuffer.h

```diff
@@ -39,6 +39,7 @@
 #include "MacroAssemblerCodeRef.h"
 #include <wtf/DataLog.h>
 #include <wtf/FastMalloc.h>
+#include <wtf/NeverDestroyed.h>
```

Comment:
Seems like this include should be in `DebugHeap.h`, since that's where the macro is defined.